fastlane/lanes/build.rb

```diff
@@ -311,98 +314,76 @@
     new_config = Xcodeproj::Config.new(versions)
     new_config.save_as(Pathname.new(version_config_path))
 
+    # Build
     gym(
-      scheme: 'Jetpack',
+      scheme: scheme,
       workspace: WORKSPACE_PATH,
-      configuration: 'Release-Alpha',
+      configuration: configuration,
       clean: true,
       output_directory: BUILD_PRODUCTS_PATH,
-      output_name: 'Jetpack Alpha',
+      output_name: output_app_name,
       derived_data_path: DERIVED_DATA_PATH,
       export_team_id: ENV.fetch('INT_EXPORT_TEAM_ID', nil),
       export_method: 'enterprise',
       export_options: { **COMMON_EXPORT_OPTIONS, method: 'enterprise' }
     )
 
+    # Upload to App Center
+    commit = ENV.fetch('BUILDKITE_COMMIT', 'Unknown')
+    pr = ENV.fetch('BUILDKITE_PULL_REQUEST', nil)
+    release_notes = <<~NOTES
+      - Branch: \`#{ENV.fetch('BUILDKITE_BRANCH', 'Unknown')}\`\n
+      - Commit: [#{commit[0...7]}](https://github.tumblr.net/#{GITHUB_REPO}/commit/#{commit})\n
+      - Pull Request: [\##{pr}](https://github.tumblr.net/#{GITHUB_REPO}/pull/#{pr})\n
```

Comment:
CI is [failing](https://buildkite.com/automattic/wordpress-ios/builds/12233#01862849-e296-495a-86fb-a6e6edca0ad9/843-4670) with 

```
lanes/build.rb:336:in `build_and_upload_installable_build': \e[31m[!] uninitialized constant Fastlane::FastFile::GITHUB_REPO\e[0m (NameError)
```

I opened #20072 to address the issue in `trunk`.